cephci/utils/configure.py

```diff
@@ -277,15 +296,25 @@ def add_images_to_private_registry(node, username, password, registry, build_typ
         f"{DOMAIN_CERT_PATH_CERT}:/certs/domain.cert:Z",
     ]
 
-    src_creds = f"{username}:{password}"
+    # Get login credentials
+    con = get_cephci_config()
+    src_username = con.get("cdn_credentials", {}).get("username")
+    src_password = con.get("cdn_credentials", {}).get("password")
```

Comment:
Same as above, this is restricting library to live server. Pass all these details through function parameter

Reply:
Sure, I'll pass these through a test case.